indico/modules/events/abstracts/controllers/management_test.py

```diff
@@ -0,0 +1,59 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2021 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+import pytest
+
+from indico.modules.events.abstracts.models.abstracts import Abstract
+from indico.modules.events.abstracts.models.review_questions import AbstractReviewQuestion
+from indico.modules.events.abstracts.models.review_ratings import AbstractReviewRating
+from indico.modules.events.abstracts.models.reviews import AbstractAction, AbstractReview
+from indico.modules.events.abstracts.settings import abstracts_reviewing_settings
+from indico.modules.events.tracks import Track
+
+
+@pytest.fixture
+def dummy_abstract(db, dummy_event, dummy_user):
```

Comment:
Maybe this should be in a more central place (`indico/modules/abstracts/testing/fixtures.py`) since we now have two `dummy_abstract` fixtures?